### PR TITLE
privacy policy changes from Adam Bier

### PR DESCRIPTION
**Pull request by jadeqwang**
Privacy Shield requirements (including separating out U.S.-Swiss and U.S.-E.U.) and various other edits.

### CODE REVIEW

File: oasis-privacy.md [hunk 1]
```diff
@@ -79,24 +83,39 @@ The [W3C "Do Not Track" standard](http://www.w3.org/TR/tracking-dnt/) defines "t
...
+
+In compliance with the EU-US Privacy Shield Principles, Sandstorm commits to resolve complaints about your privacy and our collection or use of your personal information.  European Union individuals with inquiries or complaints regarding this privacy policy should first contact Sandstorm at: privacy@sandstorm.io
+ 
+Sandstorm has further committed to refer unresolved privacy complaints under the EU-US Privacy Shield Principles BBB EU PRIVACY SHIELD, a non-profit alternative dispute resolution provider located in the United States and operated by the Council of Better Business Bureaus. If you do not receive timely acknowledgment of your complaint, or if your complaint is not satisfactorily addressed, please visit www.bbb.org/EU-privacy-shield/for-eu-consumers/ for more information and to file a complaint.
```
Comment (ocdtrekkie) [inline oasis-privacy.md:103]: Something language-y seems to be missing before BBB EU PRIVACY SHIELD. It doesn't read right. Like "to"?

**Comment by ocdtrekkie** [comment]
Also, I won't make individual line notes for all of them, but the old text made URLs and emails into links as appropriate, and the new text does not. For ease of navigation, this should be fixed.

Two additional notes:
- Doesn't the effective date also need to be changed to 9/2, since it isn't as of yet posted on Oasis?
- Is it reasonable to suggest that policy changes be (briefly) open for review before change?

**Comment by kentonv** [comment]
@ocdtrekkie This PR was merged instantly by mistake; normally it would have been open for review. That said, I don't think there's much subject to community review in this particular change -- the text is essentially what we're required to write for EU Privacy Shield compliance.

I linkified the text per your suggestion and updated the effective date to today (since I actually only just pushed it to oasis.sandstorm.io/privacy today). Thanks for the suggestions.

**Comment by ocdtrekkie** [comment]
Thanks

**Pull request merged**